### PR TITLE
add missing commands to the getting started guide

### PR DESCRIPTION
**Pull request by Nicolas-Roberto**
Update "Create your first SecretStore" and "Create your first ExternalSecret" topics to be easy to understand

## Problem Statement

The getting started guide missing two commands in  "Create your first SecretStore" and "Create your first ExternalSecret" topics, the command is `kubectl apply -f my-file.yaml`

## Related Issue

N/A

## Proposed Changes

Add command `kubectl apply -f my-file.yaml` where is missing

## Checklist

- [X] I have read the [contribution guidelines](https://external-secrets.io/latest/contributing/process/#submitting-a-pull-request)
- [X] All commits are signed with `git commit --signoff`
- [ ] (N/A) My changes have reasonable test coverage
- [ ] (N/A) All tests pass with `make test`
- [ ] (N/A) I ensured my PR is ready for review with `make reviewable`

### CODE REVIEW

File: docs/introduction/getting-started.md [hunk 1]
```diff
@@ -49,16 +49,46 @@ kubectl create secret generic awssm-secret --from-file=./access-key --from-file=
 
 ### Create your first SecretStore
 
+Create a file 'basic-secret-store.yaml' using `nano` or your preferred editor by running the following command. 
+
+```
+nano basic-secret-store.yaml
```
Comment (shuheiktgw) [inline docs/introduction/getting-started.md:55]: It would be better if we could create a file using the `echo` since that is more OS agnostic 🙂

**Review by shuheiktgw — COMMENTED** [review]
Thanks for your PR, @Nicolas-Roberto! I've added one minor comment, but it almost LGTM

File: docs/introduction/getting-started.md (hunk 1 above)
Comment (Nicolas-Roberto) [reply docs/introduction/getting-started.md:55]: Makes sense, I'll make a correction.  😃

Comment (Nicolas-Roberto) [reply docs/introduction/getting-started.md]: Does this [commit](https://github.com/external-secrets/external-secrets/pull/2751/commits/6c160129d655df3a09d262d1f699626717007907) solve the problem?

Comment (shuheiktgw) [reply docs/introduction/getting-started.md]: Ah, sorry I meant `cat` not `echo`... Then we don't have to use nano or any other editors.

``` bash
cat > basic-secret-store.yaml << EOF
...
EOF
```

Comment (shuheiktgw) [reply docs/introduction/getting-started.md]: Hmm, I see where you stack. We don't seem to be able to use `include` with bash commands. Why don't we just ask users to create a file without specifying an editor like below?


### Create your first SecretStore

Create a file 'basic-secret-store.yaml' with the following content.

```yaml
{ % include 'basic-secret-store.yaml' % }
```

Apply the new file to a resource by filename.

```bash
kubectl apply -f basic-secret-store.yaml
```

### Create your first ExternalSecret

Create a file 'basic-external-secret.yaml' with the following content.

```yaml
{ % include 'basic-external-secret.yaml' % }
```

Apply the new file to a resource by filename.

```bash
kubectl apply -f basic-external-secret.yaml
```

Comment (shuheiktgw) [reply docs/introduction/getting-started.md]: Also, please be ware that you need to commit with `--sign` flag!

Comment (Nicolas-Roberto) [reply docs/introduction/getting-started.md]: I committed the changes.
Thanks for the advice.

Comment (shuheiktgw) [reply docs/introduction/getting-started.md]: Thank you for fixing it! It seems the DCO job still fails. Would you fix it?

Comment (Nicolas-Roberto) [reply docs/introduction/getting-started.md]: Sorry for the delay, I followed the instructions provided for correction, was the problem resolved?

**Review by shuheiktgw — APPROVED** [review]
Thanks!

**Pull request merged**